plugins/catalog-backend/src/modules/codeowners/lib/read.ts

```diff
@@ -52,6 +52,7 @@ export async function findCodeOwnerByTarget(
   reader: UrlReader,
   targetUrl: string,
   scmIntegration: ScmIntegration,
+  pattern?: string,
```

Comment:
Not a pattern really; should be named `url` or `catalogInfoFileUrl` or something. Also interesting to see, what's the overlap between that, and the `targetUrl` above? Do they complement each other? Should one be removed?

Reply:
I went with the catalogInfoFileUrl here. Checked the targetUrl and the managed-by-location and you were right the target URL is actually enough for us.